Source/UnitTests/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+add_custom_target(unittests)
+add_custom_command(TARGET unittests POST_BUILD COMMAND ${CMAKE_CTEST_COMMAND})
+set_target_properties(unittests PROPERTIES FOLDER "Tests")
```

Comment:
Moving this closer to the actual tests makes sense, but why is this even here in the first place? Doesn't CMake automatically add a `test` (for Makefile and Ninja) or `RUN_TESTS` (for Visual Studio) target?